File: python/ZMQClient.py
#!/usr/bin/env python2.7

import zmq
import os

context = zmq.Context()
socket = context.socket(zmq.REQ)

socket.connect("tcp://localhost:5555")

pid = os.getpid()

while(True):
    socket.send(str("message from client ") + str(pid))
    msg = socket.recv()
    print pid, ":", msg

File: PHP/ZMQClient.php
<?php

//include 'zmsg.php';

$context = new ZMQContext();
$socket = new ZMQSocket($context, ZMQ::SOCKET_REQ);

$socket->connect("tcp://localhost:5555");

$pid = posix_getpid();

while(true) {
    $socket->send("message from client $pid");
    $msg = $socket->recv();
    echo "$pid : $msg";
}

?>